gapic/api/artman_iam.yaml

```diff
@@ -0,0 +1,26 @@
+common:
+  api_name: iam-v1
+  import_proto_path:
+    - ${REPOROOT}/googleapis
+  src_proto_path:
+    - ${REPOROOT}/googleapis/google/iam/admin/v1
+    - ${REPOROOT}/googleapis/google/iam/v1
```

Comment:
I don't think having the iam/v1 directory is correct - that is the mixin; we don't want to generate a GAPIC surface for it.